app/src/screenshotTest/kotlin/io/homeassistant/companion/android/settings/views/SettingsRowScreenshotTest.kt

```diff
@@ -0,0 +1,60 @@
+package io.homeassistant.companion.android.settings.views
+
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.mikepenz.iconics.typeface.library.community.material.CommunityMaterial
+
+class SettingsRowScreenshotTest {
+
+    @Preview
+    @Composable
+    fun `SettingsRow with only title and subtitle`() {
+        SettingsRow(
+            primaryText = "Title",
+            secondaryText = "Subtitle",
+            icon = null,
+            onClicked = {},
+        )
+    }
+
+    @Preview
+    @Composable
+    fun `SettingsRow with custom icon composable`() {
+        SettingsRow(
+            primaryText = "Title",
+            secondaryText = "Subtitle",
+            icon = {
+                Spacer(Modifier.size(width = 56.dp, height = 24.dp))
```

Comment:
Not really an icon ^^

Reply:
I think that for the sake of the screenshot visualization it would be nice to add a background to it, like a red one.